### ISSUE TITLE
Use $count without IEdmModel with EnableDependencyInjection

### ISSUE THREAD

**Issue by ramil-mux** [OData/WebApi#1818]
Hello!
I would like to know if I can use the $count method without creating an IEdmModel using only EnableDependencyInjection in Startup class ASP.NET Core web application and [EnableQuery] in Controller?
Using only EnableDependencyInjection and [EnableQuery] method $count=true don't work - returns only the list of records of the current object.
If I use IEdmModel and MapODataServiceRoute in Sturtup class, then the $count method works - returns the list of records of the current object and count attribute.

**Comment by flipdoubt** [comment]
I have the same **yes or no** question. Is using `IEdmModel` and `MapODataServiceRoute` required to get the full count using `$count=true`? If the answer is **yes**, then that is the approach we must take for now. If the answer is **no**, can someone point us to a working example?

**Comment by ramil-mux** [comment]
> I have the same **yes or no** question. Is using `IEdmModel` and `MapODataServiceRoute` required to get the full count using `$count=true`? If the answer is **yes**, then that is the approach we must take for now. If the answer is **no**, can someone point us to a working example?

Yes, if you use `IEdmModel` and `MapODataServiceRoute`, then `$count` will return the value. 
My question is whether `$count` can return a value without `IEdmModel` and `MapODataServiceRoute`?

**Comment by AnoopChatra** [comment]
Hello,

Any updates on this issue ?
